### PR TITLE
Taking the beaker out of a cryotube now tries to put it into your hand

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
tweak: Taking the beaker out of a cryotube now tries to put it into your hand
/:cl:

### CODE REVIEW

**Comment by YPOQ** [comment]
This will teleport the beaker to the AI core if it click the eject button.

**Comment by MoreRobustThanYou** [comment]
@YPOQ Silicons doesn't have hands, so it'll just eject it normally

**Comment by Qbopper** [comment]
if you actually tested that and it works then holy fuck, god tier pr

**Comment by GunHog** [comment]
What about someone with TK?

**Comment by praisenarsie** [comment]
borgs?

**Comment by MoreRobustThanYou** [comment]
@praisenarsie they have some snowflake hands

**Comment by MoreRobustThanYou** [comment]
Ok, i fixed it so you have to be next to the cryo cell and NOT A SILICON for it to attempt to put it in the user's hands

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -377,6 +377,8 @@
 		if("ejectbeaker")
 			if(beaker)
 				beaker.forceMove(loc)
+				if(get_dist(src, usr) <= 1 && !issilicon(usr))
```
Comment (Cyberboss) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: use Adjacent instead of get_dist

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**